src/app/styles/layout/_catalog-grid.less

```diff
@@ -1,12 +1,14 @@
 .l-product-grid {
     // use flexbox for layout on small+ screens
     // add flex display to product grid wrapper
+    margin: 0 -5px;
```

Comment:
specifically declare {margin-left:-5px;margin-right:-5px} since that is what we are overriding